packages/middleware/src/helpers/metrics.ts

```diff
@@ -0,0 +1,25 @@
+import { RequestHandler } from "express";
+import prometheusClient from "prom-client";
+
+const prometheusRegister = new prometheusClient.Registry();
+prometheusRegister.setDefaultLabels({
+  app: "middleware",
+});
+prometheusClient.collectDefaultMetrics({ register: prometheusRegister });
+
+/**
+ * Express API handler that should be registered on the /metrics route
+ * so that Prometheus can periodically call the middleware's /metrics endpoint
+ * and scrape the metrics
```

Comment:
What kind of metrics? This endpoint will be open (no token or whatever), anyone can hit it. Is it safe to expose such data?